pkg/pillar/kubeapi/kubeapi.go

```diff
@@ -0,0 +1,320 @@
+// Copyright (c) 2024 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build kubevirt
```

Comment:
Do we need to to add this line for all kube specific files ?

Reply:
For all files that import k8s packages. You can grep "go:build kubevirt" in Naiming's repo to see where it will be used.